src/hyperloglog.c

```diff
@@ -1009,7 +1009,7 @@ uint64_t hllCount(struct hllhdr *hdr, int *invalid) {
     double m = HLL_REGISTERS;
     double E;
     int j;
-    double alphaInf = 0.5 / log(2.);
+    static double alphaInf = 0.5 / log(2.);
```

Comment:
This will compile on GCC but not clang:

```bash
hyperloglog.c:1015:34: error: initializer element is not a compile-time constant
    static double alphaInf = 0.5 / log(2.);
```